google_symptoms/delphi_google_symptoms/geo.py

```diff
@@ -64,7 +64,7 @@ def geo_map(df, geo_res):
     for _date in df["timestamp"].unique():
         val_lists = df[df["timestamp"] == _date].merge(
                 map_df["geo_id"], how="right"
-                )[METRICS + ["combined_symptoms"]].fillna(0)
+                )[METRICS + ["sum_anosmia_ageusia"]].fillna(0)
```

Comment:
Could you extract this string to a constant somewhere? That way future changes won't have to be as arduous.

Reply:
Done.